scss/core/_functions.scss

```diff
@@ -31,3 +31,116 @@
 
   @return $base-color;
 }
+
+//
+// This will convert a hex value as string and will return a
+// decimal as number
+// This function only works with a hex string number of two digits,
+// like "A8", "B5" etc.
+//
+@function hexToDec($hex) {
```

Comment:
Function nomenclature in sccss would be camelcase like js or kebabcase like others `@functions`.
https://github.com/eduNEXT/paragon/pull/1/files#diff-8194e86e43ab5038fe99bab75594d3fbea6954734c15a35b7754d106c1cf6b5cR71